pages/your_tasks/real_dev_environments.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Fast, Declarative, Reproducible and Composable Developer Environments using Nix 
+description: "Going a step beyond just \"virtual envirnments\" - with advantages and drawbacks"
+contributors:
+  - "Nicole Dresselhaus"
+page_id: developer_environments
+related_pages:
+  your_tasks: 
+    - virtual_environments
+---
+
+## How are devenvs different from virtual software development environments?
+
+Developer Environments (short {% tool "devenv" %}s) are a more holistic
+approach to the classical virtual environments. Virtual environments - despite
+their undisputed advantages - tend to break at similar steps in the real world.
+An environment working on one machine might break on another because of
+different operating system, the existence or absence of globally installed
+packages, different hardware etc.
+You local software might behave different as a docker-container, yield different
+results or may not run at all.
```

Comment:
This paragraph nicely explains why virtual envs do not always suffice, but I miss a brief description of what a devenv itself actually is and how that solves the virtual envs' problems.